### PR TITLE
Move 390 compiles from ubuntu16 to Redhat7

### PR DESCRIPTION
**Pull request by jdekonin**
* [skip ci]
* eclipse/openj9#7791

Signed-off-by: Joe deKoning <joe_dekoning@ca.ibm.com>

### CODE REVIEW

**Comment by jdekonin** [comment]
Need this PR to merge and be deployed first.
https://github.com/eclipse-cbi/jiro/pull/84

**Comment by pshipton** [comment]
jenkins test sanity zlinux jdk11

**Comment by jdekonin** [comment]
As @AdamBrousseau pointed out, `sw.os.redhat` is incorrect it should be `sw.os.rhel`.  And then `sw.os.rhel` and `sw.os.rhel.7` for the machine labels.  I'll update both PRs this morning.

File: buildenv/jenkins/variables/defaults.yml [hunk 1]
```diff
@@ -193,9 +193,9 @@ s390x_linux:
     8: 'linux-s390x-normal-server-release'
     11: 'linux-s390x-normal-server-release'
   node_labels:
-    build: 'ci.role.build && hw.arch.s390x && sw.os.ubuntu'
+    build: 'ci.role.build && hw.arch.s390x && sw.os.rhel'
```
Comment (AdamBrousseau) [inline buildenv/jenkins/variables/defaults.yml]: Should probably make this specifically `sw.os.rhel.7`. In the event we add rhel8 for example.

**Comment by jdekonin** [comment]
jenkins test sanity zlinux jdk11

**Pull request merged**